xscen/extract.py

```diff
@@ -506,8 +506,9 @@ def search_data_catalogs(
     match_hist_and_fut: bool = False,
     periods: list = None,
     id_columns: Optional[List[str]] = None,
-    allow_resampling: bool = True,
-    allow_conversion: bool = True,
+    allow_resampling: bool = False,
+    allow_conversion: bool = False,
+    conversion_yaml: str = None,
```

Comment:
What do you think about accepting `Union[bool, str]` for  `allow_conversion`? So if it is True, it uses the default file and if it is a str/path, it uses that file? 
It's ok if you think this is confusing.

Reply:
Yeah, I fear that this might end up being more confusing than is worth :/